src/core/ext/filters/client_channel/lb_policy/weighted_target/weighted_target.cc

```diff
@@ -45,7 +45,6 @@
 #include "src/core/lib/debug/trace.h"
 #include "src/core/lib/gprpp/debug_location.h"
 #include "src/core/lib/gprpp/orphanable.h"
-#include "src/core/lib/gprpp/ref_counted.h"
```

Comment:
I think there's a lot of unrelated changes...

Reply:
IWYU sometimes makes strange changes, not sure why.